addons/html_builder/static/src/website_blog/blog_post_page_option.xml

```diff
@@ -22,15 +22,15 @@
             <BuilderCheckbox actionParam="{views: ['website_blog.opt_blog_readable']}"/>
         </BuilderRow>
         <BuilderRow label.translate="Sidebar">
-            <BuilderCheckbox actionParam="{views: ['website_blog.opt_blog_sidebar_show']}"/>
+            <BuilderCheckbox id="'blog_posts_sidebar_opt'" actionParam="{views: ['website_blog.opt_blog_sidebar_show']}"/>
         </BuilderRow>
-        <BuilderRow label.translate="Archives">
+        <BuilderRow label.translate="Archives" t-if="isActiveItem('blog_posts_sidebar_opt')" level="1">
             <BuilderCheckbox actionParam="{views: ['website_blog.opt_sidebar_blog_index_archives']}"/>
         </BuilderRow>
-        <BuilderRow label.translate="Follow Us" level="1">
+        <BuilderRow label.translate="Follow Us" t-if="isActiveItem('blog_posts_sidebar_opt')" level="1">
             <BuilderCheckbox actionParam="{views: ['website_blog.opt_sidebar_blog_index_follow_us']}"/>
         </BuilderRow>
-        <BuilderRow label.translate="Tags List" level="1">
+        <BuilderRow label.translate="Tags List" t-if="isActiveItem('blog_posts_sidebar_opt')" level="1">
```

Comment:
Just a detail. I prefer to have one <t t-if="isActiveItem('blog_posts_sidebar_opt')">.
it s easier to read.